metron-platform/metron-management/README.md

```diff
@@ -19,11 +19,72 @@ project.
 
 The functions are split roughly into a few sections:
 * Shell functions - Functions surrounding interacting with the shell in either a nicer way or a more functional way.
+* File functions - Functions around interacting with local or HDFS files
 * Configuration functions - Functions surrounding pulling and pushing configs from zookeeper
 * Parser functions - Functions surrounding adding, viewing, and removing Parser functions.
 * Enrichment functions - Functions surrounding adding, viewing and removing Stellar enrichments as well as managing batch size and index names for the enrichment topology configuration
 * Threat Triage functions - Functions surrounding adding, viewing and removing threat triage functions.
 
+### File Functions
+
+* Local Files
+  * `FILE_LS`
```

Comment:
Regarding all of the HDFS/FILE functions: How about we change wording a bit? 
FILE_GET/PUT -> LOCAL_READ/WRITE
HDFS_GET/PUT -> HDFS_READ/WRITE

Rationale - make explicit the FS that the reads/writes are using. I had considered simply passing an arg to a unified function (i.e., making FILE_READ/WRITE handle local and hdfs), but then you lose autocomplete. The existing division of labor is more explicit. Also, the GET/PUT names have some terminology overlap with the hdfs shell commands that have different results, e.g. hdfs dfs -get/put operate on files, not file contents. READ/WRITE helps keep a distinction.